src/rebar3.erl

```diff
@@ -272,4 +272,14 @@ start_and_load_apps() ->
     application:start(asn1),
     application:start(public_key),
     application:start(ssl),
-    inets:start().
+    inets:start(),
+    inets:start(httpc, [{profile, hex}]),
```

Comment:
Ah we don't use it for resources, nevermind that point. I still think we should use the profile there.